proto/src/prost/cosmos.auth.v1beta1.rs

```diff
@@ -120,6 +120,7 @@ pub mod query_client {
         ) -> QueryClient<InterceptedService<T, F>>
         where
             F: tonic::service::Interceptor,
+            T::ResponseBody: Default,
```

Comment:
...and there are some minor changes in these bounds also everywhere in the client bindings. It's not too drastic, I believe, and looks like a backward compatible relaxation.